cmd/cluster/encryption/encryption.go

```diff
@@ -135,4 +191,6 @@ func init() {
 	For GCP:
 	cloud-provider=GCP,gcp-resource-id=<resource-id>,gcp-service-account-path=<service-account-path>.`)
 	updateCmk.MarkFlagRequired("encryption-spec")
+	updateCmkState.Flags().Bool("enable", false, "Enable EAR")
+	updateCmkState.Flags().Bool("disable", false, "Disable EAR")
```

Comment:
Agreed with @posriniv, generally we use one flag, enable or not.

Reply:
Good point @posriniv - My comments earlier on were not to have any additional arguments for enable/disable at all. It will be simply --enable, --disable with no additional sub-flags or args.

If you don't have additional flags, I think it is intuitive. Agreed with additional flags it is confusing. WDYT?

Reply:
@posriniv In the current implementation, --enable and --disable flag does not have any additional options, so it should not be confusing to the user. I have handled the edge cases and thrown appropriate error if the user gives invalid input (both or no flags are present). 

One additional point is that, update CMK state is a very important operation and having the user explicitly mention enable and disable would ensure that the user does not turn on or off EAR by mistake via the default value.

Reply:
if the current implementation looks good, should I go ahead with it?
@posriniv @kc-yb

Reply:
Okay cool, if then everyone is aligned with this approach, then I can go ahead with this.
Can you please approve the PR @posriniv ?